### DISCUSSION TITLE
What is the use of from sqlalchemy.orm import Query

### DISCUSSION THREAD

**Discussion by Abdur-rahmaanJ** [sqlalchemy/sqlalchemy#7408] [Usage Questions] [answered]
I did not find any docs on the use of Query with capital Q. Can someone explain why and when do we need to use Query? Thanks

**Comment by CaselIT** [comment]
Hi,

Query is the class of the object returned by session.query. Look for query in the documentation and you will find lots of examples

**Comment by Abdur-rahmaanJ** [comment, marked as answer]
```
q = Query([User, Address], session=some_session)
```
The above is equivalent to:
```
q = some_session.query(User, Address)
```
from [here](https://docs.sqlalchemy.org/en/14/orm/query.html#sqlalchemy.orm.Query.__init__)
